scenes/e2e/src/app.e2e-spec.ts

```diff
@@ -1,11 +1,20 @@
 import {AppPage} from './app.po';
+import {screenshot} from '../screenshot';
 
 describe('screenshot scenes for each component', () => {
   // These tests simply serve as a convenient way to take snapshots of different pages,
   // they are not actually testing anything
   let page: AppPage;
+  const components = ['input'];
 
   beforeEach(() => {
     page = new AppPage();
   });
+
+  for (const comp of components) {
+    it('screenshot for ' + comp + ' scene', async () => {
```

Comment:
nit: same comment as other PR about using backtick string (can just wait for that one to merge then rebase, accept their edit, and put 'input' back in the list above)